beacon/interface.go

```diff
@@ -4,8 +4,12 @@ import (
 	"context"
 	"time"
 
+	"github.com/libp2p/go-libp2p-core/peer"
```

Comment:
please use p2p.Peer, so that libp2p is not imported anywhere outside p2p module. if i forgot to update it somewhere that is a mistake

Reply:
done, for other packages too.